### PR TITLE
updated bug_report.md and added request.md

### PR DESCRIPTION
**Pull request by sargonas**
Added proper issue template for requests with auto-tagging of label and also added an auto-label for bugs to the bug template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/request.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: Feature Request
```
Comment (altendky) [inline .github/ISSUE_TEMPLATE/request.md:2]: Just to be consistent with the `Bug report` above.  I could go either way.  And yeah, this obviously is a nit pick... `:]`

```suggestion
name: Feature request
```

File: .github/ISSUE_TEMPLATE/request.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+about: Log a request for a new feature or enhancement
+title: ''
+labels: 'enhancement '
+assignees: ''
+
+---
+
+**Describe the request**
+A clear and concise description of your suggested enhancement, as well as some basic rationale to allow us to balance the effort/reward for the work, if possible.
```
Comment (altendky) [inline .github/ISSUE_TEMPLATE/request.md:11]:
```suggestion
<!-- A clear and concise description of your suggested enhancement, as well as some basic rationale to allow us to balance the effort/reward for the work, if possible. -->
```

File: .github/ISSUE_TEMPLATE/request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+**Images**
+If applicable (but not required), add diagrams or wireframes to help explain your request if it is UI/UX related.
```
Comment (altendky) [inline .github/ISSUE_TEMPLATE/request.md:14]:
```suggestion
<!-- If applicable (but not required), add diagrams or wireframes to help explain your request if it is UI/UX related. -->
```

File: .github/ISSUE_TEMPLATE/request.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+**Additional comments**
+Add any other comments here.
```
Comment (altendky) [inline .github/ISSUE_TEMPLATE/request.md:17]:
```suggestion
<!-- Add any other comments here. -->
```

**Review by altendky — CHANGES_REQUESTED** [review]
Thanks again @sargonas.  It's great to have help with this infrastructure work.  A few tweaks that I think make it a bit tidier.  Putting the commentary in comments I think helps avoid issues with users forgetting to delete them and thus them showing up in the actual issue.

File: .github/ISSUE_TEMPLATE/request.md (hunk 4 above)
Comment (sargonas) [reply .github/ISSUE_TEMPLATE/request.md]: All this makes sense to me!

**Review by altendky — APPROVED** [review]
Thanks again, looking forward to the organization.  `:]`

**Pull request merged**